src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -72,7 +72,7 @@ file that was distributed with this source code.
                                             {% set sortable = false %}
                                             {% if field_description.options.sortable is defined and field_description.options.sortable %}
                                                 {% set sortable             = true %}
-                                                {% set sort_parameters      = admin.modelmanager.sortparameters(field_description, admin.datagrid) %}
+                                                {% set sort_parameters      = admin.modelmanager.sortparameters(field_description, admin.datagrid)|merge({_list_mode: admin.getListMode()}) %}
```

Comment:
After the change, the name of this variable is misleading. It should either be renamed, or maybe the `merge` should be done when `sort_parameters` is used?

Reply:
we should keep sort_parameters and introduce a new `parameters` below where we merge `sort_parameters` and the list mode, WDYT @greg0ire

Reply:
I don't think there is a need for a new variable. Wouldn't `admin.generateUrl('list', sort_parametersmerge({_list_mode: admin.getListMode()})) ` be ok?

Reply:
Yes if it's not used somewhere else in template 👍

Reply:
It's OK for me!